fvm/crypto.go

```diff
@@ -135,13 +135,18 @@ func verifySignatureFromRuntime(
 	sigAlgo := RuntimeToCryptoSigningAlgorithm(signatureAlgorithm)
 	if sigAlgo == crypto.UnknownSigningAlgorithm {
 		return false, errors.NewValueErrorf(signatureAlgorithm.Name(), "signature algorithm type not found")
-
+	}
+	if sigAlgo == crypto.BLSBLS12381 {
+		return false, errors.NewValueErrorf(signatureAlgorithm.Name(), "signature algorithm type %s not supported", crypto.BLSBLS12381.String())
 	}
 
 	hashAlgo := RuntimeToCryptoHashingAlgorithm(hashAlgorithm)
 	if hashAlgo == hash.UnknownHashingAlgorithm {
 		return false, errors.NewValueErrorf(hashAlgorithm.Name(), "hashing algorithm type not found")
 	}
+	if hashAlgo == hash.KMAC128 {
```

Comment:
It is the same here as `RuntimeToCryptoHashingAlgorithm` does not support `KMAC`, feel free to keep or remove this check.

Reply:
lets keep it for now then.